src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -34,6 +37,13 @@ const ProgressTracker = ({
   labelsPosition,
   ...rest
 }) => {
+  if (!deprecatedForwardRefWarnTriggered) {
+    deprecatedForwardRefWarnTriggered = true;
+    Logger.deprecate(
+      "The `orientation` and `direction` props in `ProgressTracker` component are deprecated and will soon be removed."
```

Comment:
I'm not sure of the background here as to why these are being deprecated, but if there's an alternative prop that can do the same or any other way to achieve it, we should explain how to do that. (I'm not sure that there is but it's going to be worse than a usual breaking change if we're literally removing the ability to achieve particular functionality.)

Reply:
The DS have decided we are no longer supporting this functionality